src/gui/shader.cpp

```diff
@@ -79,11 +87,17 @@ load_shaders(
             vertex_shader_id, info_log_length, NULL, vertex_shader_error_message.data()
         );
 
-        LOG_WARNING(logger, "Vertex shader error: {}", vertex_shader_error_message);
+        LOG_WARNING(
+            logging::opengl_logger, "Vertex shader error: {}",
+            vertex_shader_error_message
+        );
     }
 
     // Compile Fragment Shader
-    LOG_DEBUG(logger, "Compiling fragment shader {}", fragment_file_path);
+    LOG_DEBUG(
+        logging::opengl_logger, "Compiling fragment shader {}",
+        fragment_file_path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.